ee/clickhouse/queries/funnels/base.py

```diff
@@ -356,8 +356,7 @@ def _get_inner_event_query(
                 extra_join = self._get_cohort_breakdown_join()
             else:
                 values = self._get_breakdown_conditions()
-                if values:
-                    self.params.update({"breakdown_values": values})
```

Comment:
Is it possible to get a test case that fails for this?

Reply:
Hmmm fair question! 🤔